pyccl/neutrinos.py

```diff
@@ -49,12 +51,19 @@ def Omeganuh2(a, m_nu, T_CMB=None):
     return OmNuh2
 
 
-def nu_masses(OmNuh2, mass_split, T_CMB=None):
+@deprecated(Omega_nu_h2)
+@warn_api()
+def Omeganuh2(a, *, m_nu, T_CMB=None):
+    return Omega_nu_h2(a, m_nu=m_nu, T_CMB=T_CMB)
+
+
+@warn_api(pairs=[("Om_nu_h2", "OmNuh2")])
+def nu_masses(*, Om_nu_h2, mass_split, T_CMB=None):
```

Comment:
DISCUSSION POINT 5

Reply:
Because the first argument is `Neutrino energy density at z=0 times h^2` which I think is not at all obvious. So the first argument has to be kw-only. Then, `mass_split` is easier to understand, but it's after `Om_nu_h2`, so it can't be positional.